CODEOWNERS

```diff
@@ -160,3 +160,5 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
 /*/extensions/filters/common/local_ratelimit @mattklein123 @rgs1
 # HTTP Kill Request
 /*/extensions/filters/http/kill_request @qqustc @htuch
+# Rate limit expression descriptor
+/*/extensions/descriptors/expr @kyessenov @lizan
```

Comment:
@lizan Added as a code owner. Let me know if that doesn't work for you.

Reply:
ack, though I think `extensions/descriptors` are too generic, can we have a better name?

Reply:
We can spell it out as `rate_limit_descriptors`. Would that be better?